snapshot-labs/sx-monorepo
# feat: show transaction progress modal after vote

### Summary



Toward: #386

For onchain proposals only, this PR will open the newly added TransactionProgress modal while sending and confirming the vote transaction.

### How to test

1. Vote on an onchain proposal
2. It should close the vote modal, then open the transaction progress modal when you cast a vote
3. When the tx is confirmed, it should auto-close the transaction progress modal (we're skipping the success message, as we're showing the sharing modal in a later PR)
4. Vote on an offchain proposal
5. It should just close the vote modal after you cast a vote

### To-Do

- [x] To merge after #610
